### DISCUSSION TITLE
我真的要疯了！

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#233] [General]
@boomzero @PythonSmall-Q

**Comment by langningchen** [comment]
你们俩看看我发的朋友圈，我真的快疯了！
![](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/81672884/45615188-c389-4d12-9696-f44f22511dfb)

**Comment by langningchen** [comment]
:angry:

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q

**Comment by PythonSmall-Q** [comment]
我看到了orz
